api/python/arrus/utils/probe_check.py

```diff
@@ -363,40 +438,60 @@ def __get_signal_duration(self, rf: np.ndarray) -> float:
         return round(3 * sigma)
 
 
-class ProbeElementValidator(ABC):
+class FootprintSimilarityPCCExtractor(ProbeElementFeatureExtractor):
```

Comment:
Also, unittests for FootprintSimilarityPCCExtractor are missing.

Reply:
done